proposed/event-dispatcher-meta.md

```diff
@@ -3,7 +3,7 @@ Event Dispatcher Meta Document
 
 ## 1. Summary
 
-The purpose of this document is to describe the rationale and logic behind the Event Dispatcher PSR.
+The purpose of this document is to describe the rationale and logic behind the Event Dispatcher specification.
```

Comment:
While I know we are defining PSRs, they are more generally _specifications_, and using that verbiage here makes the document more approachable for those not familiar with the PSR system in my opinion.